### PR TITLE
Interpreter bitwise op

### PR DESCRIPTION
**Pull request by ratik21**
- Added classes for `%`, `|`, `&`, `^`, `~`
- Added tests

### CODE REVIEW

File: packages/algorand-js/test/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -412,4 +413,187 @@ describe("Teal Opcodes", function () {
...
+    const stack = new Stack<StackElem>();
+
+    it("should return correct modulo of two unit64", function () {
+      stack.push(BigInt("2"));
+      stack.push(BigInt("5"));
+      const op = new Mod();
+      op.execute(stack);
+
+      const top = stack.pop();
+      assert.equal(top, BigInt("1"));
```
Comment (robert-zaremba) [inline packages/algorand-js/test/src/interpreter/opcode-list.ts:427]: Please add tests for:
* (7, 7)


Can be in the same unit-test (same "it")

File: packages/algorand-js/test/src/interpreter/opcode-list.ts [hunk 2]
```diff
@@ -412,4 +413,187 @@ describe("Teal Opcodes", function () {
...
+    });
+
+    it("should throw error if bitwise-xor is used with strings", function () {
+      stack.push(toBytes("str1"));
+      stack.push(toBytes("str2"));
+      const op = new BitwiseXor();
+      expectTealError(
+        () => op.execute(stack),
+        ERRORS.TEAL.INVALID_TYPE
+      );
```
Comment (robert-zaremba) [inline packages/algorand-js/test/src/interpreter/opcode-list.ts:565]: for checking failure on strings, you can have a function and use it in tests multiple times:
```
function execExpectError(stack, strs: string[],op: Op, err) {
  return () => {
    for (let s of strs) stack.push(s)
    expectTealError(() => op.execute(stack), err)
  }
}

it("should ...", execExpectError("str1", "str2", newe BitwiseXor(), ERRORS.TEAL.INVALID_TYPE)
```

And you can reuse it in few places above.

**Review by robert-zaremba — APPROVED** [review]

File: packages/algorand-js/test/src/interpreter/opcode-list.ts (hunk 1 above)
Comment (ratik21) [reply packages/algorand-js/test/src/interpreter/opcode-list.ts:427]: ok

File: packages/algorand-js/test/src/interpreter/opcode-list.ts (hunk 2 above)
Comment (ratik21) [reply packages/algorand-js/test/src/interpreter/opcode-list.ts:565]: Yes. Thanks

**Pull request merged**